.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,109 @@
+id: 

```

Comment:
@jpobst: this feels "weird"/"wrong" to me.

First, it's an empty value for "id", which just feels weird.  Secondly, none of the examples use `id:` *at all*, e.g. https://microsoft.github.io/GitOps/policies/examples/pr-triage.html

Should we remove `id:`?

Reply:
This is what the creators of the service are proposing.  I think we should defer to them that they know what is best. 😁